### PR TITLE
Bug 1923081: Improve test for checking common templates

### PR DESCRIPTION
**Pull request by gouyang**
Verified in https://cnv-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/test-kubevirt-[console-4](https://issues.redhat.com/browse/console-4).7-cnv-2.6/31/console

### CODE REVIEW

**Comment by rawagner** [comment]
/lgtm cancel
failing build

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts [hunk 1]
```diff
@@ -70,10 +71,21 @@ describe('Create VM from Template using wizard', () => {
...
+      .setName(TemplateByName.RHEL7)
+      .setProvisionSource(ProvisionSource.URL)
+      .build();
+
+    const vmtName = await vmTemplate.getResourceName();
+    const workload = vmtName.split('-')[1];
+    const flavor = vmtName.split('-')[2];
+
+    await browser.wait(until.presenceOf(templateView.defaultOSLabel));
+    expect(await templateView.workload(`openshift-${vmtName}`).getText()).toContain(workload);
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts:85]:
```suggestion
    expect(await (templateView.workload(`openshift-${vmtName}`) as any).getText()).toContain(workload);
```

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts [hunk 2]
```diff
@@ -70,10 +71,21 @@ describe('Create VM from Template using wizard', () => {
...
+    const detailFlavor = await templateView.flavor(`openshift-${vmtName}`).getText();
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts:86]:
```suggestion
    const detailFlavor = await (templateView.flavor(`openshift-${vmtName}`) as any).getText();
```

Comment (gouyang) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vmtemplate.wizard.scenario.ts]: @yaacov Thanks, it works in this way.

**Pull request merged**